Model/ProcessOrder/Processing.php

```diff
@@ -71,10 +77,26 @@ public function execute(OrderInterface $order, array $rvvupData): ProcessOrderRe
 
             $order = $this->changeNewOrderStatus($order);
 
+            $eventMessage = 'Rvvup Payment is being processed.';
+
+            $payment = $rvvupData['payments'][0];
+            if ($this->inAuthorizedManualPayment($payment)) {
+                $formattedExpirationDate = $this->dateTime->date(
+                    "jS M Y H:i:s T",
+                    strtotime($payment["authorizationExpiresAt"])
+                );
+
+                $eventMessage = "Payment authorization expires at " .
+                    $formattedExpirationDate .
+                    ". Please navigate to the Rvvup dashboard to manually capture the payment. " .
+                    "When the authorization expires, the order will be cancelled" .
```

Comment:
Tiny text issue here as theres no space at the end.
Results in this display
```
... be cancelledand the funds ...
```

So should be 
```
"When the authorization expires, the order will be cancelled " .
```

Reply:
<img width="562" alt="Screenshot 2023-12-06 at 09 38 10" src="https://github.com/rvvup/magento-plugin/assets/96384911/96ce4c3a-bf8e-4bf8-922e-b0b9dd78292e">

Also noticed it has 
```
Rvvup Payment ID: OR01HGZ7BYJNG5FZZAEMMXWDCJ2B
``` 

Should this be the PA... id rather than the OR order id?

Reply:
Agree, there is separate card for that, in scope of which it will be done. Nice work!